README.md

```diff
@@ -1,4 +1,8 @@
 # workflows
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/SciTools/workflows/main.svg)](https://results.pre-commit.ci/latest/github/SciTools/workflows/main)
 
-Reusable GitHub workflows to be called from other `SciTools` repositories.
+Common [reusable GitHub workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows) to be called from other `SciTools` repositories.
+
+## License
+
+`workflows` is distributed under the terms of the [BSD-3-Clause](https://spdx.org/licenses/BSD-3-Clause.html) license.
```

Comment:
TBH I went with that because as a developer it's good to know the SPDX short-identifier string for the license... perhaps not the best motivation, but there you go

Reply:
I asked because I wasn't familiar with SPDX. I've had a bit of a look into it. From what I can tell it is a standard for communicating the licence (and other stuff) so it seems fine to add here.
My concern was whether the use of it claimed we were fully adopting the standard and what that would mean, but I suppose it's no bad thing to do that